mobile/params.go

```diff
@@ -71,8 +71,8 @@ func GoerliGenesis() string {
 // FoundationBootnodes returns the enode URLs of the P2P bootstrap nodes operated
 // by the foundation running the V5 discovery protocol.
 func FoundationBootnodes() *Enodes {
-	nodes := &Enodes{nodes: make([]*enode.Node, len(params.MainnetBootnodes))}
-	for i, url := range params.MainnetBootnodes {
+	nodes := &Enodes{nodes: make([]*enode.Node, len(params.EtherumMainnetBootnodes))}
```

Comment:
I don't know if we can use it in the future or not. But this should be change to Ronin bootnodes instead, shouldn't it?

Reply:
It's backward compatibilities in running testing right now, so we will keep it first.